src/Schema/Events/Conversations.ts

```diff
@@ -0,0 +1,46 @@
+import { ActionType } from "."
+
+/**
+ * Schemas describing Conversation events
+ * @packageDocumentation
+ */
+
+/**
+ * A user focused on message input on a conversation
+ *
+ * This schema describes events sent to Segment from [[focusedOnConversationMessageInput]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "focusedOnConversationMessageInput",
+ *    impulse_conversation_id: 198
+ *
+ *  }
+ * ```
+ */
+export interface FocusedOnConversationMessageInput {
+  action: ActionType.focusedOnConversationMessageInput
+  impulse_conversation_id?: number
```

Comment:
@anipetrov Do we also need a new context module or it's fine not supplying that for now?

Reply:
no i dont think we need one!